api/ShadowRoot.json

```diff
@@ -15,41 +15,33 @@
           },
           "edge": {
             "version_added": false,
-            "notes": "Under consideration"
+            "notes": "<a href='https://developer.microsoft.com/microsoft-edge/platform/status/shadowdom/'>Under consideration</a>"
           },
           "edge_mobile": {
             "version_added": false,
-            "notes": "Under consideration"
+            "notes": "<a href='https://developer.microsoft.com/microsoft-edge/platform/status/shadowdom/'>Under consideration</a>"
           },
           "firefox": {
             "version_added": "59",
             "flags": [
-              {
```

Comment:
@ExE-Boss Why have you removed this flag? Where did you find that it doesn't command the activation of this API>

Reply:
The `dom.webcomponents.enabled` flag has been removed in Firefox 59 ([bug 1428685](https://bugzil.la/1428685)).